src/ui/source_editor_area.ui

```diff
@@ -23,18 +23,44 @@
    <property name="spacing">
     <number>0</number>
    </property>
-   <property name="margin">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
-    <widget class="QPlainTextEdit" name="editor">
-     <property name="backgroundVisible">
-      <bool>false</bool>
+    <widget class="edbee::TextEditorWidget" name="edbeeEditorWidget" native="true">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="font">
```

Comment:
Do you _have_ to specify a font here - if you omit this won't it _inherit_ the parent widget's font - or _are_ you looking to override that here?